cli/src/semdep/parse_lockfile.py

```diff
@@ -97,6 +101,8 @@ def parse_dep(s: str) -> FoundDependency:
     "maven_dep_tree.txt": None,
     "gradle.lockfile": "build.gradle",
     "pnpm-lock.yaml": None,
+    "packages.lock.json": None,
+    "packages.config": "packages.config",
```

Comment:
What's this?

Reply:
oops I added this temporarily, and forgot to remove it.